### PR TITLE
CloudWatch: small logs refactors

### LINKED ISSUE #44073: CloudWatch: Small logs code refactors
- Remove angular.copy call in datasource.ts
- Move hardcoded value to a const at the top of log_actions.go
- executeLogAlertQuery calls executeStartQuery, then calls alertQuery which also calls executeStartQuery, and the first is redundant
- Move alertQuery const variables to the top of the file

part of #44072

### PR DESCRIPTION
**Pull request by iwysiu**
**What this PR does / why we need it**:
Minor refactors for cloudwatch logs (mostly moving constants) Iisted in the linked issue

**Which issue(s) this PR fixes**:

Fixes #44073

**Special notes for your reviewer**:

### CODE REVIEW

File: public/app/plugins/datasource/cloudwatch/datasource.ts [hunk 1]
```diff
@@ -123,7 +122,7 @@ export class CloudWatchDatasource
   }
 
   query(options: DataQueryRequest<CloudWatchQuery>): Observable<DataQueryResponse> {
-    options = angular.copy(options);
```
Comment (sarahzinger) [inline public/app/plugins/datasource/cloudwatch/datasource.ts:126]: 👏 is this the last place we are using angular here?

File: pkg/tsdb/cloudwatch/cloudwatch.go [hunk 2]
```diff
@@ -322,13 +324,6 @@ func (e *cloudWatchExecutor) executeLogAlertQuery(ctx context.Context, req *back
 			return nil, err
 		}
 
-		result, err := e.executeStartQuery(ctx, logsClient, model, q.TimeRange)
```
Comment (sarahzinger) [inline pkg/tsdb/cloudwatch/cloudwatch.go:325]: do you mind talking through this one more? Why did we have it/why remove it?

**Review by sarahzinger — APPROVED** [review]
cool cool

File: pkg/tsdb/cloudwatch/cloudwatch.go (hunk 2 above)
Comment (iwysiu) [reply pkg/tsdb/cloudwatch/cloudwatch.go:325]: the alertQuery function below calls executeStartQuery on the same model and replaces the QueryId with the one from that response, so there's not point to calling it here

File: public/app/plugins/datasource/cloudwatch/datasource.ts (hunk 1 above)
Comment (iwysiu) [reply public/app/plugins/datasource/cloudwatch/datasource.ts:126]: yup! 😄

File: pkg/tsdb/cloudwatch/cloudwatch.go (hunk 2 above)
Comment (sarahzinger) [reply pkg/tsdb/cloudwatch/cloudwatch.go:325]: ah gotcha! thanks!

File: public/app/plugins/datasource/cloudwatch/datasource.ts (hunk 1 above)
Comment (sarahzinger) [reply public/app/plugins/datasource/cloudwatch/datasource.ts:126]: cool!

**Pull request merged**